tasks.rb

```diff
@@ -784,6 +784,11 @@ def set_hpxml_climate_and_risk_zones(hpxml_file, hpxml)
     hpxml.climate_and_risk_zones.weather_station_name = 'Duluth, MN'
     hpxml.climate_and_risk_zones.weather_station_epw_filepath = 'USA_MN_Duluth.Intl.AP.727450_TMY3.epw'
     hpxml.header.state_code = 'MN'
+  elsif ['base-location-helena-mt.xml'].include? hpxml_file
+    hpxml.climate_and_risk_zones.iecc_zone = '6B'
```

Comment:
You might consider using a method to return this zone, like I'm using for the BuildResHPXML measure: https://github.com/NREL/OpenStudio-HPXML/blob/06d3ad74b243767c8afa2bfc8acd51ba5eeac41e/BuildResidentialHPXML/resources/location.rb#L13